src/App.js

```diff
@@ -47,13 +42,18 @@ class App extends Component {
 
 
   }
+  
+  handleChange = event => {
+    this.setState({
+      searchQuery: event.target.value,
+    });
+  }
 
   render() {
     return (
       <Container>
         <Form>
-          <Form.Label>Enter your search..</Form.Label>
-          <Form.Control onChange= {this.handleChange} value={this.state.searchQuery}/>
+        <input onChange={(event) => this.setState({ searchQuery: event.target.value })} placeholder="search for a city"></input>
```

Comment:
Feedback here:

I left a comment regarding the setState call in render in previous PR. It looks like you did in fact create a handler for the event, but yet, we still have this setState call here. You will want to remove this (see reasons stated for this in previous PR :) )